supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystem.scala

```diff
@@ -33,91 +34,24 @@ object SharedFileSystem {
   }
 
   case class PairOfFiles(src: File, dst: File)
-  type DuplicationStrategy = (File, File) => Try[Unit]
-
-  /**
-    * Return a `Success` result if the file has already been localized, otherwise `Failure`.
-    */
-  private def localizePathAlreadyLocalized(originalPath: File, executionPath: File): Try[Unit] = {
-    if (executionPath.exists) Success(()) else Failure(new RuntimeException(s"$originalPath doesn't exists"))
-  }
-
-  private def localizePathViaCopy(originalPath: File, executionPath: File): Try[Unit] = {
-    executionPath.parent.createDirectories()
-    val executionTmpPath = pathPlusSuffix(executionPath, ".tmp")
-    Try(originalPath.copyTo(executionTmpPath, overwrite = true).moveTo(executionPath, overwrite = true)).void
-  }
-
-  private def localizePathViaHardLink(originalPath: File, executionPath: File): Try[Unit] = {
-    executionPath.parent.createDirectories()
-    Try { executionPath.linkTo(originalPath, symbolic = false) } void
-  }
-
-  private def localizePathViaSymbolicLink(originalPath: File, executionPath: File): Try[Unit] = {
-      if (originalPath.isDirectory) Failure(new UnsupportedOperationException("Cannot localize directory with symbolic links"))
-      else {
-        executionPath.parent.createDirectories()
-        Try { executionPath.linkTo(originalPath, symbolic = true) } void
-      }
-  }
-
-  private def duplicate(description: String, source: File, dest: File, strategies: Stream[DuplicationStrategy]) = {
-    import cromwell.util.FileUtil._
 
-    strategies.map(_ (source.followSymlinks, dest)).find(_.isSuccess) getOrElse {
-      Failure(new UnsupportedOperationException(s"Could not $description $source -> $dest"))
-    }
-  }
+  type DuplicationStrategy = (File, File) => Try[Unit]
 
   def pathPlusSuffix(path: File, suffix: String) = path.sibling(s"${path.name}.$suffix")
 }
 
-trait SharedFileSystem extends PathFactory {
+trait SharedFileSystem extends PathFactory with StrictLogging {
   import SharedFileSystem._
   import better.files._
 
-  def sharedFileSystemConfig: Config
```

Comment:
OK, np... I will revert those changes and just keep it to minimal. It will not look good but if this helps, there is no problem from my side.